src/uct/sm/mm/xpmem/configure.m4

```diff
@@ -28,13 +28,17 @@ AS_IF([test "x$with_xpmem" != "xno"],
               ])
        ])
 
-# Verify XPMEM header file
+# Verify XPMEM lib and header files
 AS_IF([test "x$xpmem_happy" = "xno" -a -d "$with_xpmem"],
-      [AC_CHECK_HEADER([$with_xpmem/include/xpmem.h],
+      [AC_CHECK_LIB([$with_xpmem/lib/xpmem],
```

Comment:
in addition, since AC_CHECK_LIB overrides LDFLAGS, need to save and restore them.
for example: https://github.com/openucx/ucx/blob/master/config/m4/cuda.m4#L49

Reply:
Fixed now.

Reply:
in order to make it self-contained, can you pls move `SAVE_LDFLAGS="$LDFLAGS"` and `LDFLAGS="$SAVE_LDFLAGS"` inside AS_IF, right before and after AC_CHECK_LIB?

Reply:
Done. Have one more look please. Thanks!